TeamSnapSDK/TSDKTeamSnap.m

```diff
@@ -184,6 +184,11 @@ - (void)rootLinksWithConfiguration:(TSDKRequestConfiguration *)configuration com
             
                 [self.rootLinks getSchemasWithConfiguration:configuration completion:schemaCompletionBlock];
             } else {
+                TSDKCollectionObject *cachedObject = [TSPCache objectOfClass:[TSDKRootLinks class] withId:@""];
+                if([cachedObject isKindOfClass:[TSDKRootLinks class]]) {
+                    weakSelf.rootLinks = [TSPCache objectOfClass:[TSDKRootLinks class] withId:@""];
```

Comment:
Why are we not just using the `cachedObject` var we just unpacked from the cache above if it is valid?

Reply:
Updated